saleor/graphql/checkout/schema.py

```diff
@@ -29,10 +33,11 @@
 
 
 class CheckoutQueries(graphene.ObjectType):
-    checkout = graphene.Field(
+    checkout = FieldWithChannel(
         Checkout,
-        description="Look up a checkout by token.",
+        description="Look up a checkout by token and slug of channel.",
         token=graphene.Argument(UUID, description="The checkout's token."),
+        slug=graphene.Argument(graphene.String, description="Slug of the checkout"),
```

Comment:
Wrong description.

Reply:
We now have `FieldWithChannel` which adds `channel` arg and the `slug` arg which seems like not needed?